### ISSUE TITLE
KeyError when I try to run Randomizer version 1.1.0

### ISSUE THREAD

**Issue by marmot7777** [Gonzalooo/Scarlet-and-Violet-Randomizer#17]
I am getting a KeyError when I try to run Randomzier version 1.1.0.  It may be user error, but here it is.

`Traceback (most recent call last):
  File "C:\Users\User\Documents\Switch Games\1_Pokemon\Randomizer\Scarlet-and-Violet-Randomizer-main\Randomizer.py", line 315, in <module>
    randomize()
  File "C:\Users\User\Documents\Switch Games\1_Pokemon\Randomizer\Scarlet-and-Violet-Randomizer-main\Randomizer.py", line 212, in randomize
    randomize_based_on_config(config)
  File "C:\Users\User\Documents\Switch Games\1_Pokemon\Randomizer\Scarlet-and-Violet-Randomizer-main\Randomizer.py", line 156, in randomize_based_on_config
    paldea_fight, kita_fight, blue_fight = TrainerRandomizer.randomize_trainers(config)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Switch Games\1_Pokemon\Randomizer\Scarlet-and-Violet-Randomizer-main\Randomizer\Trainers\trainerrando.py", line 842, in randomize_trainers
    make_poke(config['paldea_settings']['trainers_randomizer']['raid_npc_randomizer'],
  File "C:\Users\User\Documents\Switch Games\1_Pokemon\Randomizer\Scarlet-and-Violet-Randomizer-main\Randomizer\Trainers\trainerrando.py", line 672, in make_poke
    if config['force_6_pokemon'] == "yes":
       ~~~~~~^^^^^^^^^^^^^^^^^^^
KeyError: 'force_6_pokemon'
Press any key to continue . . .`

[new_config.json](https://github.com/Gonzalooo/Scarlet-and-Violet-Randomizer/files/14971249/new_config.json)

**Comment by Gonzalooo** [comment]
Should be fixed on 1.1.0-Beta-2

**Comment by Gonzalooo** [comment]
@marmot7777 could you please make sure that it is fixed on 1.1.0-beta-2. If its not fixed like I thought it was lmk so I can fix it

**Comment by marmot7777** [comment]
@Gonzalooo , appears to be working! I was able to run the randomizer and even tested it out in the game. Thank you! Can't tell you how much joy this randomizer brings my kids and her friends.
